README.md

```diff
@@ -2,6 +2,59 @@
 
 [![Node.js CI](https://github.com/facebookexperimental/Recoil/workflows/Node.js%20CI/badge.svg)](https://github.com/facebookexperimental/Recoil/actions)
 
-Recoil is an experimental set of utilities for state management with React. Please see the website: https://recoiljs.org
+Recoil is an experimental set of utilities for state management with React.
+
+Recoil lets you create a data-flow graph that flows from atoms (shared state) through selectors (pure functions) and down into your React components. Atoms are units of state that components can subscribe to. Selectors transform this state either synchronously or asynchronously.
+
+## Installation
+
+The Recoil package lives in <a href="https://www.npmjs.com/get-npm" target="_blank">npm</a>. To install the latest stable version, run the following command:
```

Comment:
I think markdown link syntax is better than the HTML link.

Reply:
Thank you for your suggestion
I have replaced all HTML links by markdown link syntax as following
## Installation

The Recoil package lives in [npm](https://www.npmjs.com/get-npm). To install the latest stable version, run the following command:

```shell
npm install recoil
```

Or if you're using [yarn](https://classic.yarnpkg.com/en/docs/install/)

```shell
yarn add recoil
```